appserver/docker/src/main/java/fish/payara/docker/instance/admin/CreateDockerContainerCommand.java

```diff
@@ -127,36 +208,36 @@ public void execute(AdminCommandContext adminCommandContext) {
                 dasHost, dasPort);
 
         // Create web target with query
-        Client client = ClientBuilder.newClient();
-        WebTarget webTarget = null;
-        if (Boolean.valueOf(node.getUseTls())) {
-            webTarget = client.target("https://"
-                    + node.getNodeHost()
-                    + ":"
-                    + node.getDockerPort()
-                    + "/containers/create");
-        } else {
-            webTarget = client.target("http://"
-                    + node.getNodeHost()
-                    + ":"
-                    + node.getDockerPort()
-                    + "/containers/create");
-        }
+        WebTarget webTarget = createWebTarget(node, "/containers/create");
         webTarget = webTarget.queryParam(DockerConstants.DOCKER_NAME_KEY, instanceName);
 
         // Send the POST request
         Response response = null;
         try {
-            response = webTarget.queryParam(DockerConstants.DOCKER_NAME_KEY, instanceName)
-                    .request(MediaType.APPLICATION_JSON).post(Entity.entity(jsonObject, MediaType.APPLICATION_JSON));
+            response = webTarget.request(MediaType.APPLICATION_JSON).post(Entity.entity(jsonObject, MediaType.APPLICATION_JSON));
         } catch (Exception ex) {
             logger.log(Level.SEVERE, "Encountered an exception sending request to Docker: \n", ex);
         }
 
         // Check status of response and act on result
         if (response != null) {
             Response.StatusType responseStatus = response.getStatusInfo();
-            if (!responseStatus.getFamily().equals(Response.Status.Family.SUCCESSFUL)) {
+            if (responseStatus.getFamily().equals(Response.Status.Family.SUCCESSFUL)) {
+                // Read the container ID from the response
+                JsonObject jsonResponse = response.readEntity(JsonObject.class);
+
+                if (jsonResponse != null) {
+                    String dockerContainerId = jsonResponse.getString("Id");
```

Comment:
`Id` key defined in lower case on StopDockerContainerCommand @RestParam(name="id") annotation.

Reply:
This is referencing the "Id" field from the Docker Rest API, not our own.
The Docker Rest API has the capital :)
https://docs.docker.com/engine/api/v1.39/#operation/ContainerCreate